polygon/heimdall/testutil/test_json_marshal.go

```diff
@@ -0,0 +1,21 @@
+package testutil
```

Comment:
sharing a convention ive seen used for this - `heimdalltest` package for having heimdall specific test utils

Reply:
given this func is not used outside of the heimdall package there is maybe no need for a separate testutil package at this point (maybe in the future), so you can use another trick which is to add this to a file `polygon/heimdall/test_util_test.go`

Reply:
test_util_test.go didn't work, see: https://stackoverflow.com/questions/31794141/can-i-create-shared-test-utilities
I will rename the package